js/content/common.js

```diff
@@ -212,6 +212,18 @@ let ExtensionLayer = (function() {
         script.parentNode.removeChild(script);
     };
 
+    // NOTE: use cautiously!
+    // Load remote script in the context of the current tab
+    self.loadInPageContext = function(fun) {
+        return new Promise(function(res) {
+            let script  = document.createElement("script");
+            script.src = fun;
+            script.onload = res;
```

Comment:
Why no handler for `script.onerror`?
Network errors could prevent the script from loading and there is certain to be a time in the future when these script URLs change. We should degrade gracefully in those circumstances.